config/application.rb

```diff
@@ -1,5 +1,26 @@
 require_relative 'boot'
 
+require 'logger'
+
+# Copy the google analtyics key to a local JSON file.
+# This code runs before rails itself is loaded.
+
+
+logger = Logger.new(STDOUT)
+logger.level = Logger::DEBUG
```

Comment:
Looks like you can delete the logger stuff since we're not logging anything anymore.

Reply:
done